### DISCUSSION TITLE
CLK or SCK. Typo?

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#351] [Q&A] [answered]
Quick question:

- [This link](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck-pin-allocation) (see table header) uses "CLK" as one of the three SPI channels
- [This link](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#expansion-port-pinout) (see the image) uses "SCK" as one of the three SPI channels

Is this a typo, or am I misunderstanding? If the answer is "CLK", does this stand for "clock"? Thanks!

**Comment by krichardsson** [comment, marked as answer]
Yes, they are both the SPI clock. 
The SPI clock is usually called CLK or SCK for Serial Clock

**Reply by krichardsson** [reply]
I will update the terminology on the web to use SCK in the table as well. Thanks for pointing it out!

**Reply by Pandagineer** [reply]
Thank you. Serial clock: good to know
